src/Symfony/Component/HttpFoundation/Session/Flash/FlashBag.php

```diff
@@ -165,16 +163,4 @@ public function clear()
     {
         return $this->all();
     }
-
-    /**
-     * Returns an iterator for flashes.
-     *
-     * @deprecated Will be removed in 3.0.
-     *
-     * @return \ArrayIterator An \ArrayIterator instance
-     */
-    public function getIterator()
-    {
```

Comment:
Can you submit a PR to add a deprecation warning when used in 2.7? And the Changelog should also be updated in 2.7. Thanks.